jest.config.js

```diff
@@ -1,30 +1,66 @@
 const { jestDirAlias } = require('./dirAlias');
 
 module.exports = {
-  reporters: [
-    'default',
-    [
-      'jest-junit',
-      {
-        suiteName: 'Jest Tests',
-        outputDirectory: 'reports/jest',
-        uniqueOutputName: 'true',
-        ancestorSeparator: ' › ',
+  projects: [
+    {
+      displayName: 'Unit Tests',
+      reporters: [
+        'default',
+        [
+          'jest-junit',
+          {
+            suiteName: 'Unit Tests',
+            outputDirectory: 'reports/jest',
+            uniqueOutputName: 'true',
+            ancestorSeparator: ' › ',
+          },
+        ],
+      ],
+      collectCoverageFrom: [
+        '**/(src|scripts)/**/*.{js,jsx}',
+        '!**/src/testHelpers/**',
+        '!**/*.stories.jsx',
+        '!**/src/integration/**/*.{js,jsx}',
+      ],
+      setupFiles: ['./src/testHelpers/jest-setup.js'],
+      testMatch: [
+        '**/__tests__/**/*.js?(x)',
+        '**/?(*.)+(spec|test).js?(x)',
+        '!**/src/integration/**/*.{js,jsx}',
+      ],
+      snapshotSerializers: ['enzyme-to-json/serializer'],
+      setupFilesAfterEnv: ['./src/testHelpers/setupTests.js'],
+      moduleNameMapper: jestDirAlias,
+      transform: {
+        '^.+\\.js$': 'babel-jest',
+        '^.+\\.jsx$': 'babel-jest',
       },
-    ],
-  ],
-  collectCoverageFrom: [
-    '**/(src|scripts)/**/*.{js,jsx}',
-    '!**/src/testHelpers/**',
-    '!**/*.stories.jsx',
+    },
+    {
+      displayName: 'Integration Tests',
+      testEnvironment: './src/integration/integrationTestEnvironment.js',
+      reporters: [
+        'default',
+        [
+          'jest-junit',
+          {
+            suiteName: 'Integration Tests',
+            outputDirectory: 'reports/jest',
+            uniqueOutputName: 'true',
+            ancestorSeparator: ' › ',
+          },
+        ],
+      ],
+      collectCoverageFrom: ['**/src/integration/**/*.{js,jsx}'],
```

Comment:
What does the coverage mean for integration tests? If it is just testing against the DOM of a given page it cannot know what lines of code have been executed and covered?

Reply:
This is true. Been meaning to remove this to avoid confusion 👍